lib/Providers/Qr/BaconQrCodeProvider.php

```diff
@@ -23,10 +22,10 @@ class BaconQrCodeProvider implements IQRCodeProvider
      * Ensure we using the latest Bacon QR Code and specify default options
      */
     public function __construct(
-        private int $borderWidth = 4,
+        private readonly int $borderWidth = 4,
         private string|array $backgroundColour = '#ffffff',
         private string|array $foregroundColour = '#000000',
-        private string $format = 'png',
+        private string       $format = 'png',
```

Comment:
I can appreciate the nice visuals of having all the variables line up but the practice in general usually ends up with needing to micro-manage spaces on unrelated lines when making changes so I'd say this wasn't worth it

Reply:
It wasn't me who did this, though, it was the PHP-CS-Fixer.